test/functional/wallet_assumeutxo.py

```diff
@@ -185,6 +192,14 @@ def run_test(self):
         self.log.info("Ensuring wallet can be restored from backup")
         n1.restorewallet("w", "backup_w.dat")
 
+        # Now make sure background validation succeeds on n2
+        # and try restoring the wallet
+        self.connect_nodes(0, 2)
+        self.sync_blocks(nodes=(n0, n2))
+        self.wait_until(lambda: len(n2.getchainstates()['chainstates']) == 1)
+        self.log.info("Ensuring wallet can be restored from backup (pruned node)")
```

Comment:
nit: You could move this to the top of this block like with the other cases (unless you duplicate anyway)

Reply:
Done